### PR TITLE
accessibilty: added accessibilty fields to the mobile edit button

### PR DESCRIPTION
**Pull request by lpranam**
Change-Id: Ifdb59ff3463a0354549316994f8e80b341e28133

* Resolves: # 
* Target version: master 

### Checklist

- [ ] Code is properly formatted
- [ ] All commits have Change-Id
- [ ] I have run tests with `make check`
- [ ] I have issued `make run` and manually verified that everything looks okay
- [ ] Documentation (manuals or wiki) has been updated or is not required

### CODE REVIEW

File: browser/src/control/Permission.js [hunk 1]
```diff
@@ -13,6 +13,9 @@ L.Map.include({
 	setPermission: function (perm) {
 		var button = $('#mobile-edit-button');
 		button.off('click');
+		button.attr('tabindex', 0);
+		button.attr('role', 'button');
+		button.attr('title', 'Enter editing');
```
Comment (pedropintosilva) [inline browser/src/control/Permission.js:18]: title is for the tooltip. Which yup is nice to have but we should also add alt attribute

**Review by pedropintosilva — CHANGES_REQUESTED** [review]

File: browser/src/control/Permission.js (hunk 1 above)
Comment (lpranam) [reply browser/src/control/Permission.js:18]: alt attribute does not exist for div technically. but should I add it anyway? image of the pencil is actually a div with the background.

Comment (pedropintosilva) [reply browser/src/control/Permission.js:18]: aha so if it's not a button we should either:
1. Turn that thing into a button (and fix all the styles and things that will brake with that change)
2. Or for now use aria attributes https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA

Comment (lpranam) [reply browser/src/control/Permission.js:18]: its already ARIA compatible for for button role: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Comment (lpranam) [reply browser/src/control/Permission.js]: done

**Review by pedropintosilva — APPROVED** [review]

**Pull request merged**